src/atomate2_temp/common/jobs/phonons.py

```diff
@@ -18,29 +18,120 @@
     CubicSupercellTransformation,
 )
 
-from pymatgen.phonon.bandstructure import PhononBandStructureSymmLine
-from pymatgen.phonon.dos import PhononDos
+from atomate2_temp.aims.jobs.phonons import (
```

Comment:
Seems to be not a very good practice to let a common job depend on code-specific; may be some polymorphism can be employed here?

Reply:
easier to just not create a default phonon_maker and make it a non-optional variable